lib/core/utils/preload-media.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Given a rootNode
+ * -> get all HTMLMediaElement's and ensure their metadata is loaded
+ *
+ * @method preloadMedia
+ * @memberof axe.utils
+ * @property {Object} options.treeRoot (optional) the DOM tree to be inspected
+ */
+axe.utils.preloadMedia = function preloadMedia({ treeRoot = axe._tree[0] }) {
```

Comment:
This should have a timeout option. Users need to be able to specify how long they want to wait for this.

Reply:
Each of the preload(able) items do not have a special timeout. As in there is a global timeout specified on the `preload` itself, over specifying individual timeouts for each item. 

[This is how we timeout now](https://github.com/dequelabs/axe-core/pull/1958/files/27c18bd54f051116fcf4710c21f17ccc89687b0c#diff-34951a6fe41c0b8543bc30e1c882fe9aR100)

Example we cannot say timeout is 10000 for CSSOM and 2000 for MEDIA. We can simple say all preload expires in a given timeout.

Reply:
Thanks for the clarification.